scenes/src/app/scenes/radio/radio-scene.html

```diff
@@ -0,0 +1,8 @@
+<div class="scene-radio-title">
+  <mat-icon class="scene-radio-icon">language</mat-icon>
+</div>
```

Comment:
I think this scene would look fine without this, but I don't feel strongly either way.

Reply:
+1 I would prefer not to have the icon